ledger/accountdb.go

```diff
@@ -422,17 +422,17 @@ func accountsDbInit(r db.Queryable, w db.Queryable) (*accountsDbQueries, error)
 	var err error
 	qs := &accountsDbQueries{}
 
-	qs.listCreatablesStmt, err = r.Prepare("SELECT asset, creator FROM assetcreators WHERE asset <= ? AND ctype = ? ORDER BY asset desc LIMIT ?")
+	qs.listCreatablesStmt, err = r.Prepare("SELECT rnd, asset, creator FROM acctrounds LEFT JOIN assetcreators ON assetcreators.asset <= ? AND assetcreators.ctype = ? WHERE acctrounds.id='acctbase' ORDER BY assetcreators.asset desc LIMIT ?")
```

Comment:
Why not just `SELECT .. FROM acctrounds, assetcreators ON ...`?  This would be a cross-join in SQLite, so that if there are no matching rows in the `assetcreators` table, this would return no rows as before.  I think that would require fewer special cases for dealing with the resulting row set, compared to what you're currently handling in this PR.

Reply:
I'm not sure if that's what we want here. We *want* to receive the database round even when there are not rows, since a mismatching round number between the database and the internal data structure means we need to re-evaluate.

Reply:
Ah, I see, yeah, you're right.  Good point!